staging/src/k8s.io/apimachinery/pkg/semanticrv/rvbytes.go

```diff
@@ -0,0 +1,161 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package semanticrv
+
+import (
+	"bytes"
+	"strings"
+)
+
+// ResourceVersion nominally identifies a closed set of write transactions
```

Comment:
what is nominally and what is closed?

Reply:
I have force-pushed a revision that hopefully makes the answer clearer.